templates/federated_reporting/distributed_cleanup.py.mustache

```diff
@@ -138,7 +138,7 @@ def interactive_setup(force_interactive=False):
             prompt="Enter admin password for superhub {}: ".format(socket.getfqdn())
         )
 
-    api = NovaApi(api_user="admin", api_password=admin_pass)
+    api = NovaApi(api_user="admin", api_password=admin_pass, hostname={{{vars.sys.fqhost}}}:{{{vars.cfe_internal_hub_vars.https_port}}})
```

Comment:
Ah. Sorry @aleksandrychev I don't think we need this change. The NovaApi() is used both for the superhub itself, which this change would work for, but also for feeders which this change will not work for.

I think what we need is the port information from the feeder when we attach a feeder through the UI or via API.

If someone changes the port on a feeder after already connecting to superhub, well, manual changes might be required.

Reply:
oki, removed. I will mention this in the blog post